cmake/configs/default.cmake

```diff
@@ -199,6 +199,7 @@ hunter_config(poly2tri VERSION 1.0.0)
 hunter_config(polyclipping VERSION 4.8.8-p0) # for Assimp
 hunter_config(presentproto VERSION 1.0)
 hunter_config(pthread-stubs VERSION 0.3)
+hunter_config(pugixml VERSION 1.8.1)
```

Comment:
Doesn't belong here. Only `.travis.yml`/`appveyor.yml` patches. See README:
* https://github.com/ingenue/hunter

You can find some useful info in this issue:
* https://github.com/ruslo/hunter/issues/743